src/main/java/duke/task/Event.java

```diff
@@ -0,0 +1,41 @@
+package duke.task;
+
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Task type event.
+ */
+public class Event extends Task {
+
+    private final LocalDateTime at;
```

Comment:
Perhaps a more intuitive or descriptive variable name could be used here as well?